mta/mta.sw.yaml

```diff
@@ -187,5 +189,28 @@ states:
     stateDataFilter:
       output: '{workflowOptions: {migrationOptions: [{id: "m2k", name: "Move2Kube workflow"}]}, repositoryURL: .repositoryURL}'
     end: true
-
+  - name: NotifyFailureBackstage
+    type: operation
+    actions:
+      - condition: .backstageUser != null
+        functionRef:
+          refName: createNotification
+          arguments:
+            title: "MTA analysis failed"
+            message: '"MTA analysis for " + .application.repository.url + " failed. Task pod: " + .taskgroup.pod +" ; Error: "+ .taskgroup.error'
+            origin: "MTA Analysis Workflow"
+            topic: "MTA Analysis Workflow"
+            targetUsers:
+              - .backstageUser
+      - condition: .backstageGroup != null
+        functionRef:
+          refName: createNotification
+          arguments:
+            title: "MTA analysis failed"
+            message: '"MTA analysis for " + .application.repository.url + " failed. Task pod: " + .taskgroup.pod +" ; Error: "+ .taskgroup.error'
+            origin: "MTA Analysis Workflow"
+            topic: "MTA Analysis Workflow"
+            targetGroups:
+              - .backstageGroup
+    end: true
```

Comment:
the ui plugin can deal with an assessment returning no options structue or we need an empty structure?

Reply:
Good question, I'll test without sending notification to have the workflow reach the end successfully

Reply:
The system not care if there is not output, the state is completed and no link is displayed
![image](https://github.com/parodos-dev/serverless-workflows/assets/3036508/d1155b8b-23dc-4e9b-abd3-c9fa1282ea13)